### ISSUE TITLE
All the controls should be accessible by keyboard and Calendar shortcut keys should work properly

### ISSUE THREAD

**Issue by SanjayBhargavS** [Eonasdan/tempus-dominus#2461]
Accessibility of the datetimepicker can be improved by providing keyboard navigation.

Here are few examples of Keyboard navigation:
- When a user hits the tab key, the focus should move inside the dialog instead of closing it. 
- Focus should move to all the controls (next button, previous button, year/decade/century toggle button, time picker) using the tab key .
- To select the dates/months/year/decades/century via keyboard.
- Navigate with the help of keyboard in the time picker section.

**Comment by SanjayBhargavS** [comment]
Related issue: https://github.com/Eonasdan/tempus-dominus/issues/2390

**Comment by henrique-monteiro** [comment]
waiting for solution

**Comment by kikonen** [comment]
This is very important factor, since due to accessibility requirements, cannot use this widget now due to that.

**Comment by Eonasdan** [comment]
Please add your comments to #2390 so I can have all these things in one place.
